src/main/java/com/amazon/opendistroforelasticsearch/sql/parser/FieldMaker.java

```diff
@@ -312,6 +312,12 @@ public static MethodField makeMethodField(String name, List<SQLExpr> arguments,
             paramers.addAll(tempParamers);
         }
 
-        return new MethodField(finalMethodName, paramers, option == null ? null : option.name(), alias);
+        if (!builtInScriptFunction) {
```

Comment:
A little more readable by checking `builtInScriptFunction` instead of `!builtInScriptFunction` and switch logic in `if` and `else`.

Reply:
agree, fixed